lib/utils/strings.js

```diff
@@ -84,7 +84,7 @@ exports.string2buf = function (str) {
 // Helper (used in 2 places)
 function buf2binstring(buf, len) {
   // use fallback for big arrays to avoid stack overflow
-  if (len < 65537) {
+  if (len < 65534) {
```

Comment:
This number looks magical now. Please add comment (why nobody should try to increase it back to 65536+1).

Reply:
@puzrin done.